Chanjo2 integration howto (#4573)
Clinical-Genomics/scout: docs/admin-guide/chanjo_coverage_integration.md

# Considerations

> Scout has been supporting integration with the Chanjo software for several years now.

Reviewer: Suggest either generalising to non-temporal statement, or anchoring it to particular years. Documents tend to go unedited for extended time periods, and things like "recent" only briefly is.

Author: Non-temporal is good, will do!

> Scout has been supporting integration with the Chanjo software for several years now.

Reviewer: ```suggestion
Scout is well integrated with the Chanjo software for coverage reporting.
```

> While Chanjo has been efficient and speedy in calculating metrics over transcript data, making it a valuable tool for visualizing statistics in WES analyses, the incorporation of support for other types of analyses (WGS, WTS) in Scout has rendered **Chanjo outdated**.

Reviewer: ```suggestion
While Chanjo is efficient and speedy in calculating metrics over transcript data, making it a valuable tool for visualizing statistics in WES analyses, the incorporation of support for other types of analyses (WGS, WTS) in Scout has rendered **Chanjo outdated**. In particular, updating the reference transcript set in an active Chanjo instance is challenging.
```

> In response to this, recent efforts at Clinical Genomics have led to the development of Chanjo2, an updated tool designed to calculate coverage and coverage completeness statistics with the advantage of utilizing the **.d4 (Dense Depth Data Dump) format**.

Reviewer: ```suggestion
In response to this, efforts at Clinical Genomics led to the development of Chanjo2, an updated tool designed to calculate coverage and coverage completeness statistics with the advantage of utilizing the **.d4 (Dense Depth Data Dump) format**.
```

For further information on d4tools and the .d4 format, please visit the relevant [project pages][d4tools].
Instructions on setting up a Chanjo2 server can be found in the documentation available on the [Chanjo2 GitHub pages][chanjo2].

> For system administrators configuring communication with either of these tools, **we recommend integrating with Chanjo2, as the Chanjo repository is likely to be deprecated in the near future**.

Reviewer: ```suggestion
For system administrators configuring communication with either of these tools, **we recommend integrating with Chanjo2, as the Chanjo repository is likely to be deprecated**.
```

Prior to loading a case into Scout, it's essential to create .d4 files. Instructions on how to generate these files using the d4tools software can be found in the [software's documentation][d4tools_create].

The path to the .d4 files should be provided in the .yaml Scout load config, as outlined in the [scout documentation](load-config.md).

## Chanjo coverage integration (soon deprecated)

Scout may be configured to visualize coverage reports produced by [Chanjo][chanjo].
To do so it is necessary to install the python package [Chanjo][chanjo] and its report visualization app, [chanjo-report][chanjo-report].

Both these programs require a database to store samples names, transcript specifications and statistics relative to the trascript coverage for all samples. Note that chanjo can be used as a standalone but chanjo-report needs to have access to the data produced by chanjo to generate coverage reports. **Scout does not require chanjo installed in the path to visualize coverage reports, but it is necessary to install chanjo-report and set up the connection to a database containing coverage data in order to use this fuctionality**.

> ### Chanjo database

Reviewer: 👍
